src/main/src/main/java/org/vfny/geoserver/util/Requests.java

```diff
@@ -121,6 +122,9 @@ public static boolean isLoggedIn(HttpServletRequest request) {
      * @return The InputStream used to validate and parse the SLD xml.
      */
     public static InputStream getInputStream(URL url) throws IOException {
+        // handle security for http/https
+        if (url.getProtocol().toLowerCase().startsWith("http")) URLCheckers.evaluate(url);
```

Comment:
Do not limit to http (unless there is a reason that I'm missing?). E.g., one might now want the server to access the local file system out of params from remote request calls, seems a legit concern no?).